packages/common/src/ui/layout/tables/components/Header/Header.tsx

```diff
@@ -69,18 +77,24 @@ export const HeaderCell = <T extends RecordWithId>({
       aria-label={String(key)}
       sortDirection={isSorted ? direction : false}
     >
-      {sortable ? (
-        <TableSortLabel
-          hideSortIcon={false}
-          active={isSorted}
-          direction={direction}
-          IconComponent={SortDescIcon}
-        >
+      <Tooltip
```

Comment:
Richard was asking to reduce the column headers down to the abbreviations.. and so I thought a long version would be good to have as a tooltip